connectors/source-aws-billing/README.md

```diff
@@ -98,7 +98,7 @@ docker run -it --rm \
 
 Make sure the `target` value in your config file is `http://localhost:31081` so that the Source can send CloudEvents to our Display Sink.
 
-then you view events like: 
+Here is the sort of CloudEvent you should expect to receive in the Display Sink: 
```

Comment:
This sentence is bizarre to me. Was this from the template? Maybe we should update it?  @xdlbdy @JieDing